website/docs/r/event_orchestration_global.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
This whole markdown file is heavily based off of `event_ochestration_service.html.markdown`